app/views/development_metrics/defect_escape_rate/_values_details.erb

```diff
@@ -18,7 +18,11 @@
         <h5>Bugs by environment:</h5>
         <% metric[:values].each do |metric| %>
```

Comment:
Instead of doing `metric.first` and `metric.second`, we can define this loop like:

`metric[:values].each do |environment, bugs|`

It's easier to follow

Reply:
Nice, done!